src/components/Display.js

```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types';
+
+function Display({ result }) {
```

Comment:
- [ ] [OPTIONAL] It will be nice if you use the arrow function as you are using ES6 syntax